app/navigation/router.js

```diff
@@ -148,15 +161,18 @@ class Router extends React.Component {
                         render={this.renderTransition}
                         configureTransition={this.configureTransition}
                     />
+                    <ModalOptions/>
```

Comment:
I assume this works correctly, but this actually renders the modal in the correct location even when drawers are open? I would've figured it would move the modal with the rest of the scene's contents